README.md

```diff
@@ -58,14 +58,14 @@ coin count = 3
 total balance = 0.15000000 BTC
 ```
 
-* On one terminal run the maker server with `cargo run run-maker maker.teleport`. You should see the message `listening on port 6102`.
+* On one terminal run the maker server with `cargo run -- --wallet_file_name=maker.teleport run-maker`. You should see the message `listening on port 6102`.
```

Comment:
The port number must be passed now, so `cargo run -- --wallet-file-name=maker.teleport run-maker 6102`

Reply:
Fixed in 4f57bae730516d74a7e3fe04947620dc3541afff.